modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinServerCodegen.java

```diff
@@ -268,17 +284,19 @@ public void processOpts() {
         public final static String COMPRESSION_DESC = "Adds ability to compress outgoing content using gzip, deflate or custom encoder and thus reduce size of the response.";
         public final static String LOCATIONS = "featureLocations";
         public final static String LOCATIONS_DESC = "Generates routes in a typed way, for both: constructing URLs and reading the parameters.";
+        public final static String METRICS = "featureMetrics";
+        public final static String METRICS_DESC = "Enables metrics feature.";
     }
 
     @Override
     public void postProcess() {
         System.out.println("################################################################################");
         System.out.println("# Thanks for using OpenAPI Generator.                                          #");
-        System.out.println("# Please consider donation to help us maintain this project \uD83D\uDE4F                 #");
+        System.out.println("# Please consider donation to help us maintain this project \uD83D\uDE4F                  #");
         System.out.println("# https://opencollective.com/openapi_generator/donate                          #");
         System.out.println("#                                                                              #");
         System.out.println("# This generator's contributed by Jim Schubert (https://github.com/jimschubert)#");
-        System.out.println("# Please support his work directly via https://patreon.com/jimschubert \uD83D\uDE4F      #");
+        System.out.println("# Please support his work directly via https://patreon.com/jimschubert \uD83D\uDE4F       #");
```

Comment:
Now I got the following instead with the extra space that you added:
```
################################################################################
# Thanks for using OpenAPI Generator.                                          #
# Please consider donation to help us maintain this project 🙏                  #
# https://opencollective.com/openapi_generator/donate                          #
#                                                                              #
# This generator's contributed by Jim Schubert (https://github.com/jimschubert)#
# Please support his work directly via https://patreon.com/jimschubert 🙏       #
################################################################################
```

Reply:
It looked broken on my machine before. I think it's due to the UTF-32 glyphs. I moved back to the original version.